File: Language Programing/C-programming/bubble sort.c
#include<stdio.h>
int bubble_sort();
int main()
{
    int a[5]= {7,3,12,5};
    int n,i;
        bubble_sort(a,4);
        for(i=0;i<4;i++)
            printf("%d ",a[i]);
}
int bubble_sort(int A[],int n)
{
    int round,temp,i;
    for(round=1; round<=n-1; round++)
    {
        for(i=0; i<=n-1-round; i++)
            if(A[i]>A[i+1])
            {
                temp=A[i];
                A[i]=A[i+1];
                A[i+1]=temp;
            }
    }
}

File: C#NumTheoryAlgo-STL-DS-Algo/NumberTheoryAlgo/NumberTheoryAlgo/SearchingSortingModule/BubbleSort.cs
using System;
using System.Collections.Generic;

namespace SearchingSortingModule
{
    class BubbleSort
    {
        private static List<int> BubbleSortFuncation(List<int> mylist)
        {
            for (int i = 0; i < mylist.Count - 1; i++)
            {
                for (int j = 0; j < mylist.Count - 1 - i; j++)
                {
                    if (mylist[j] > mylist[j + 1])
                    {
                        int temp = mylist[j + 1];
                        mylist[j + 1] = mylist[j];
                        mylist[j] = temp;
                    }
                }
            }
            return mylist;
        }
        public static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            var line = Console.ReadLine().Split(' ');
            var mylist = new List<int>();
            for (int i = 0; i < n; i++)
            {
                var val = int.Parse(line[i]);
                mylist.Add(val);
            }
           
            List<int> sortedList = BubbleSortFuncation(mylist);
            for(int i =0;i<mylist.Count;i++)
                Console.Write(mylist[i]+" ");
        }

        
    }
}
 
